system/CURRENT_VERSION.txt

```diff
@@ -1 +1 @@
-1.0.2
+1.0.3
```

Comment:
no need, 1.0.2 is already the current development version, 1.0.1 was last release